examples/bull-spread/bull-spread.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"cloud.google.com/go/civil"
+	"github.com/shopspring/decimal"
+
+	"github.com/alpacahq/alpaca-trade-api-go/v3/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/v3/marketdata"
+)
+
+var decimal10k = decimal.NewFromInt(10000)
+
+const (
+	oneWeek = 7
+)
```

Comment:
[minor] I don't like C-style top-level definitions. Please create these variables where they're actually used. `oneWeek` is not needed in my opinion, `AddDays(7)` is very readable.

Reply:
Removed.
7f5ef5cad60686b69fb44fc685de6a282e972c21